.github/assets/test_scripts/sftp_profile_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# This script is used to test the SFTP profile functionality of the OCI CLI.
+# It is intended to be run either from the OCI CLI's test suite or by itself, run from the base oci_env directory.
+
+ENV_FILE=".github/assets/test_scripts/sftp_compose.env"
+
+oci-env -e $ENV_FILE compose build
+oci-env -e $ENV_FILE compose up -d
+
+oci-env -e $ENV_FILE poll
+
+oci-env -e $ENV_FILE pulp file content upload --file "dev_requirements.txt" --relative-path "dev_requirements.txt"
+
+oci-env -e $ENV_FILE compose down
+rm -f test_file.txt
```

Comment:
Not sure where `test_file.txt` is coming from.

Reply:
Yeah, me neither.